samples/implicit-flow/react/package.json

```diff
@@ -7,19 +7,27 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "bootstrap": "^5.0.1",
+    "child_process": "^1.0.2",
+    "fs": "^0.0.1-security",
+    "http": "^0.0.1-security",
+    "https": "^1.0.0",
+    "https-browserify": "^1.0.0",
     "ibm-verify-sdk": "^2.0.7",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "reactstrap": "^8.9.0",
-    "web-vitals": "^1.0.1"
+    "stream-http": "^3.2.0",
+    "url": "^0.11.0",
+    "web-vitals": "^1.0.1",
+    "xmlhttprequest": "^1.8.0"
```

Comment:
Why are we adding new dependencies?

Reply:
I will try to remove it and it should be added by `npm audit fix`